lib/utils.pm

```diff
@@ -911,7 +911,8 @@ sub get_x11_console_tty {
       && !(is_leap && !leap_version_at_least('15.0'))
       && !is_sle12_hdd_in_upgrade
       && !is_caasp
-      && !get_var('VERSION_LAYERED');
+      && !get_var('VERSION_LAYERED')
+      && (check_var('VERSION', 'SLES-11'));
```

Comment:
Same as above, with sle_version_at_least('12') implemented, no change is required here.